### ISSUE TITLE
l10n_ar

### ISSUE THREAD

**Issue by lap-odoo** [odoo/odoo#18607]
Dear Odoo Partners,

At Odoo, a lot of effort is put into improving the accounting localizations for each country and also to maintain these accounting localizations up-to-date. To achieve that, we will need your help.

One github issue will be created per accounting localization (e.g. l10n_be for Belgium). We expect from you to make suggestions about what should be improved in the specific accounting localization. Do not hesitate to share files with detailed explanations.

As soon as changes will be implemented, you will be informed and will have the opportunity to test it and share your feedback.

This github issue will also be useful to keep you informed about all the ongoing improvements concerning a specific accounting localization.

Please, note that we currently don't have the resources to directly start working on l10n_ar. But we can already start to collect the information to start implementing this when we will have the resources. 

Thank you for your help

**Comment by Danisan** [comment]
The first thing to do is replace the COA by one more suitable for general purposes of the country. the AR community have been working in this during considerable time, so I think adoption of the new coa woul be an important step.
Thanks!

**Comment by lap-odoo** [comment]
@Danisan I agree that's where we should start. Which CoA do you suggest ? Could you provide a csv with the accounts and account types (or even a PR if you want) ? 

Thanks

**Comment by jjscarafia** [comment]
One requirement for argentina localization.

## More variables to evaluate taxes with "Python Code":
On Argentina there are some taxes that only apply if the invoice amount (total amount) is greater than some amount. 
We are actually using the module "account_tax_python" and configuring a tax to type "code" but on the methods for "python_compute" and "python_applicable" we don't have any variable with the information of the invoice total amount

There are also two more keys we would need when computing taxes: the partner and the date. This is because some taxes have different percentage for each partner on each month. So, in summary, to compute taxes for argentinian invoices/sale orders, we need to know:
* date of the invoice / sale order
* related partner
* total amount of the invoice/sale order

**Comment by qdp-odoo** [comment]
If I'm not wrong, you already have access to the partner (partner variable) and to the time python module. 

We will try to add the invoice data as well in the available variables.
